content/working_with/console/pages/plugins-page.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: bt_wiki
+title: Plugins Page
+category: Console
+draft: false
+weight: 155
+aliases: ["/manager_webui/plugins-page/", "/working_with/console/plugins-page/"]
+---
+
+Plugins page lists the plugins available to the current tenant.
+
+![Plugins Page]( /images/ui/pages/plugins-page.png )
+
+By default, [plugins]({{< relref "working_with/official_plugins/_index.md" >}}) are tenant-specific, meaning that a blueprint on one tenant cannot access a plugin on a different tenant. You can also set a plugin as global or private when you upload it to the manager.

```

Comment:
Tenant-specificity refers to all (or most) kind of resources we have in Cloudify, so I'm wondering do we need to include information about tenant-specificity in this page where in other pages we don't say it explicitly?

Reply:
I think a common sense would be to consider a plugin a system-wide resource, so I think this note explicitly for plugins still makes sense. For other types of resources it's not surprising that they can have different visibility.